package/skylark_download_daemon/src/skylark_download_daemon.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2017 Swift Navigation Inc.
+ * Contact: Mark Fine <mark@swiftnav.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <stdlib.h>
+#include <unistd.h>
+
+#include <libskylark.h>
+#include <sbp_zmq.h>
+
+//
+// Download Daemon - connects to Skylark and receives SBP messages.
+//
+
+// TODO (mookerji): Command line stuff?
```

Comment:
On the command line you'd probably want to pass the configuration items that are optional? e.g., the url (wouldn't pass enabled - you're running the thing you obviously want it enabled!).